windows/ReactNativeWebView2/ReactWebView2.cpp

```diff
@@ -0,0 +1,126 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+#include "pch.h"
+#include "JSValueXaml.h"
+#include "ReactWebView2.h"
+#include "ReactWebView2.g.cpp"
+#include <winrt/Windows.Foundation.Metadata.h>
+#include <optional>
+
+namespace winrt {
+    using namespace Microsoft::ReactNative;
+    using namespace Windows::Data::Json;
+    using namespace Windows::Foundation;
+    using namespace Windows::UI;
+    using namespace Windows::UI::Popups;
+    using namespace Windows::UI::Xaml;
+    using namespace Microsoft::UI::Xaml::Controls;
+    using namespace Windows::UI::Xaml::Controls;
+    using namespace Windows::UI::Xaml::Input;
+    using namespace Windows::UI::Xaml::Media;
+    using namespace Microsoft::Web::WebView2::Core;
+} // namespace winrt
+
+namespace winrt::ReactNativeWebView2::implementation {
+
+    ReactWebView2::ReactWebView2(winrt::IReactContext const& reactContext) : m_reactContext(reactContext) {
+#ifdef CHAKRACORE_UWP
```

Comment:
what's this for?

Reply:
That line was taken from the original ReactNativeWebView source. Unclear what its use case is. I can remove if you don't think it's needed.